qiling/os/windows/x8664.py

```diff
@@ -172,5 +172,5 @@ def runner(ql):
     if ql.internal_exception is not None:
         raise ql.internal_exception
 
-    ql.dprint("Syscalls and number of invocations")
-    ql.dprint(list(ql.PE.syscall_count.items()))
+    ql.dprint("[+] Syscalls and number of invocations")
```

Comment:
sorry, my bad again, i guess we move to os/windows/utils.py since these are the same code and we dont habe to call twice.

Reply:
Ok, understood what you mean

Reply:
call it post_report some something, something related to post process report and it should location before line 172, i think. looks cleaner